File: Java/BusinessApplication/app/src/main/java/com/example/mike/businessapplication/MainActivity.java
package com.example.mike.businessapplication;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity{
    private TextView txtSelected;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        this.txtSelected = (TextView) findViewById(R.id.textView);
    }

    public void changeButton(View view){
        String id = view.getTag().toString();
        //Integer id = view.getId();

        Log.i("Id tag",id);

        TextView textView = findViewById(R.id.textView);

        switch (id){
            case "3":
                Button button3 = findViewById(R.id.button3);
                textView.setText(button3.getText());
                break;
            case "2":
                Button button2 = findViewById(R.id.button2);
                textView.setText(button2.getText());
                break;
            case "1":
                Button button = findViewById(R.id.button);
                textView.setText(button.getText());
                break;
            case "4":
                Button button4 = findViewById(R.id.button4);
                textView.setText(button4.getText());
                break;
            case "5":
                Button button5 = findViewById(R.id.button5);
                textView.setText(button5.getText());
                break;
        }
    }

    public void selectButton(View view){
        if(view.getId() == R.id.button3){
            Intent i = new Intent(this, ServiciosActivity.class);
            startActivity(i);
        }else if(view.getId()==R.id.button2){
            this.txtSelected.setText("Portafolio");
        }
    }
}

File: Kotlin/Negocios/app/src/main/java/com/example/mike/negocios/MainActivity.kt
package com.example.mike.negocios

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button3.setOnClickListener {
            text.setText(button3.text)
        }

        button2.setOnClickListener {
            text.setText(button2.text)
        }

        button.setOnClickListener {
            text.setText(button.text)
        }

        button4.setOnClickListener {
            text.setText(button4.text)
        }

        button5.setOnClickListener {
            text.setText(button5.text)
        }

    }
}
